pkg/cli/admin/catalog/mirror.go

```diff
@@ -80,7 +80,7 @@ var (
 		oc adm catalog mirror file:///local/index/my/image:latest my-airgapped-registry.com
 
 		# Configure a cluster to use a mirrored registry
-		oc apply -f manifests/imageContentSourcePolicy.yaml
+		oc apply -f manifests/imageDigestMirrorSet.yaml
```

Comment:
I think also this one https://github.com/openshift/oc/blob/6e055a18a89b80c6665929185ea54a2a0133b89d/pkg/cli/admin/catalog/mirror.go#L90